.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,7 +10,7 @@
   - [ ] My commits conform to the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) specification
 - [ ] This PR is not a duplicate of a similar PR ([how to search issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/filtering-and-searching-issues-and-pull-requests) for similar/duplicate PRs, which this is not
 - [ ] This PR targets the `dev` branch
-- [ ] I ran `npm test` and all tests pass
+- [ ] I ran the pre-commit hooks or `npm test && npm lint && npm test:e2e` and all tests pass
```

Comment:
If we don't mind running an action on PRs we can probably just drop this and let the tooling test it for them. We might want to set it to run on approval though, if we do, just to be conservative with resources.

Reply:
If we are going to automate it then dropping this makes sense.